### ISSUE TITLE
I am getting strange on hover title behaviour when using with the simplest meteor/bootstrap combination. 

### ISSUE THREAD

**Issue by ficshelf** [amrali/bootstrap-select-meteor#1]
I am getting strange on hover title behaviour when using with the simplest meteor/bootstrap combination. Would like to get rid of it somehow. 

The original http://silviomoreto.github.io/bootstrap-select/3/  does not seem to produce teh same issue. 

![image](https://f.cloud.github.com/assets/5930819/2268244/2d3fad86-9ec7-11e3-8c20-26e08a2e9a04.png)

```
       <p class="lead"> 
          I would describe myself as a
          <select class="selectpicker" multiple data-selected-text-format="values" data-width="100%" title="C'mon, don't be shy! Tell us!" >
            <option data-content="<span class='label label-success'>Writer</span>" value="writer" selected >Writer</option>
            <option data-content="<span class='label label-info'>Editor</span>" value="editor" selected>Editor</option>
            <option data-content="<span class='label label-warning'>Agent</span>" value="agent">Agent</option>
            <option data-content="<span class='label label-primary'>Proofreader</span>" value="proofreader">Proofreader</option>
            <option data-content="<span class='label label-danger'>Illustrator</span>" value="illustrator">Illustrator</option>
            <option data-content="<span class='label label-default'>Reader</span>" value="reader" selected>Reader</option>
            <option data-content="<span class='label label-default'>Just a curious person!</span>" value="curious">Just a curious person!</option>
          </select>
       </p>
```

**Comment by amrali** [comment]
I'm not sure of the details, but can you post the related js (I.e,
selectpicker initialization code) and I'll take a look at it later.

FYI: You will have to {{isolate}} any reference to reactive data/functions
in the template or Meteor will rerender the DOM to the original HTML.

AA

**Comment by ficshelf** [comment]
Hi Amr

the simpleset ever:   $('.selectpicker').selectpicker();

**Comment by ficshelf** [comment]
i tried creating a project with JUST your package and the above code and nothing else +  $('.selectpicker').selectpicker();

the undesired effect is still there. Chrome/ubuntu.

no custom css/js at all/

**Comment by ficshelf** [comment]
basically your plugin produces this code:
<button type="button" class="btn dropdown-toggle selectpicker btn-default" data-toggle="dropdown" title="&lt;span class='label label-success'&gt;Writer&lt;/span&gt;, &lt;span class='label label-info'&gt;Editor&lt;/span&gt;, &lt;span class='label label-default'&gt;Reader&lt;/span&gt;"><span class="filter-option pull-left"><span class="label label-success">Writer</span>, <span class="label label-info">Editor</span>, <span class="label label-default">Reader</span></span>&nbsp;<span class="caret"></span></button>

injecting 'title' attribute which in turn shows up in chrome. Is there any reason for the title attribute or can we just modify it/remove it?

**Comment by amrali** [comment]
I'm not sure. According to docs here
(http://silviomoreto.github.io/bootstrap-select/3) it is how the title
is set on `multiple select`. So I think you are at liberty to modify
it/remove it if need be.

On 02/27/2014 03:02 PM, ficshelf wrote:

> basically your plugin produces this code:
> Writer, Editor, Reader 
> 
> injecting 'title' attribute which in turn shows up in chrome. Is there
> any reason for the title attribute or can we just modify it/remove it?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/amrali/bootstrap-select-meteor/issues/1#issuecomment-36239278.

## 

Amr

**Comment by amrali** [comment]
https://github.com/silviomoreto/bootstrap-select/issues/445

It seems that the version used on the site is older than the latest version of bootstrap-select.

I personally verified this and it does work. So just apply `$('.selectpicker').tooltip('disable');` right after `$('.selectpicker').selectpicker()`.

**Comment by ficshelf** [comment]
awesome!!

**Comment by amrali** [comment]
Considered a question, and now answered.

**Comment by GreggOD** [comment]
Hi,

I tried this but I am still getting the same issue. I even tried $('[title]').tooltip('disable') to no avail. Its a persistent little ***.

I also tried:
$('[title]').off('hover');

**Comment by GreggOD** [comment]
Hi,

I tried this but I am still getting the same issue. I even tried $('[title]').tooltip('disable') to no avail. Its a persistent little ***... and i havent had this issue with bootstrap-select before.

I also tried:
$('[title]').off('hover');
$('.selectpicker').off('hover');
$('[title]').unbind('mouseenter mouseleave');
$('.selectpicker').unbind('mouseenter mouseleave');

Im trying to debug in chrome but that keeps freezing up my browser so now im lost. And it goes with saying that the company I work for has included bootstrap like 3 times and probably jquery a few times too which seems to be giving me glitches all over... however i need to work around it.

Thanks,
G
